src/golang/lib/models/artifact.go

```diff
@@ -20,10 +20,11 @@ const (
 
 // An Artifact maps to the artifact table.
 type Artifact struct {
-	ID          uuid.UUID           `db:"id" json:"id"`
-	Name        string              `db:"name" json:"name"`
-	Description string              `db:"description" json:"description"`
-	Type        shared.ArtifactType `db:"type" json:"type"`
+	ID            uuid.UUID           `db:"id" json:"id"`
+	Name          string              `db:"name" json:"name"`
+	Description   string              `db:"description" json:"description"`
+	Type          shared.ArtifactType `db:"type" json:"type"`
+	FromLocalData bool                `db:"from_local_data" json:"from_local_data"`
```

Comment:
oh we definitely shouldn't be propagating this field to the golang side. This is an SDK-only concept.

Reply:
Make sense that this is a SDK only object. Should be addressed!